CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2220,6 +2220,150 @@ private void OnMouseButton(MouseButtonEventArgs e)
+                X = (float)touchPoint.Position.X,
+                Y = (float)touchPoint.Position.Y,
+                RadiusX = 0,
+                RadiusY = 0,
+                RotationAngle = 0,
+                PointerType = PointerType.Touch,
+                Pressure = 0,
+                Type = touchEventType,
+                Modifiers = modifiers,
+            };
+
+            browser.GetHost().SendTouchEvent(touchEvent);
+
+            e.Handled = true;
+        }
+
+        /// <summary>
+        /// Invoked when an unhandled <see cref="E:System.Windows.Input.StylusDown" /> attached event reaches an element in its route that is derived from this class. Implement this method to add class handling for this event.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.StylusDownEventArgs" /> that contains the event data.
+        protected override void OnStylusDown(StylusDownEventArgs e)
+        {
+            Focus();
+            CaptureStylus();
+            OnStylus(e, TouchEventType.Pressed);
+            base.OnStylusDown(e);
+        }
+
+        /// <summary>
+        /// Invoked when an unhandled <see cref="E:System.Windows.Input.StylusMove" /> attached event reaches an element in its route that is derived from this class. Implement this method to add class handling for this event.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.StylusDownEventArgs" /> that contains the event data.
+        protected override void OnStylusMove(StylusEventArgs e)
+        {
+            OnStylus(e, TouchEventType.Moved);
+            base.OnStylusMove(e);
+        }
+
+        /// <summary>
+        /// Invoked when an unhandled <see cref="E:System.Windows.Input.StylusUp" /> attached event reaches an element in its route that is derived from this class. Implement this method to add class handling for this event.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.StylusDownEventArgs" /> that contains the event data.
+        protected override void OnStylusUp(StylusEventArgs e)
+        {
+            ReleaseStylusCapture();
+            OnStylus(e, TouchEventType.Released);
+            base.OnStylusUp(e);
+        }
+
+        /// <summary>
+        /// Handles a <see cref="E:Stylus" /> event.
+        /// </summary>
+        /// <param name="e">The <see cref="StylusEventArgs"/> instance containing the event data.</param>
+        /// <param name="touchEventType">The <see cref="TouchEventType"/> event type</param>
+        private void OnStylus(StylusEventArgs e, TouchEventType touchEventType)
+        {
+            var stylusPoints = e.GetStylusPoints(this);
+            if (e.StylusDevice != null && e.StylusDevice.TabletDevice.Type == TabletDeviceType.Stylus && stylusPoints.Count > 0)
+            {
+                var modifiers = e.GetModifiers();
+                var pressure = stylusPoints[0].PressureFactor;
```

Comment:
Looks like you assign a `pressure` variable, doesn't appear to be used though.

Reply:
My mistake. I forgot to use the variable. Fixed.